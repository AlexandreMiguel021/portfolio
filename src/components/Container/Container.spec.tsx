import { render } from '@testing-library/react'
import { Container } from '.'
import '@testing-library/jest-dom'

describe('Container', () => {
	it('should render a Container correctly', () => {
		const { container } = render(<Container>content</Container>)
		expect(container).toMatchSnapshot()
	})
	it('should render a Container with children', () => {
		const { getByText } = render(<Container>content</Container>)
		expect(getByText('content')).toBeInTheDocument()
	})
})
