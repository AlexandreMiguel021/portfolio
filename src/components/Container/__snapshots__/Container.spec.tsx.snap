// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render a Container correctly 1`] = `
<div>
  <div
    class="container"
  >
    content
  </div>
</div>
`;
